rke/k8s_version_info.go

```diff
@@ -36,16 +36,17 @@ func loadRancherDefaultK8sVersions() map[string]string {
 		"2.6.3-patch1": "v1.21.x",
 		"2.6.3-patch2": "v1.21.x",
 		"2.6.4":        "v1.22.x",
+		"2.6.5":        "v1.23.x",
 		// rancher will use default if its version is absent
-		"default": "v1.22.x",
+		"default": "v1.23.x",
 	}
 }
 
 func loadRKEDefaultK8sVersions() map[string]string {
 	return map[string]string{
 		"0.3": "v1.16.3-rancher1-1",
 		// rke will use default if its version is absent
-		"default": "v1.22.9-rancher1-1",
+		"default": "v1.23.5-rancher1-1",
```

Comment:
We now have `v1.23.6-rancher1-1` following the merge of April 2022 k8s patches, so I think it should be updated here.

Reply:
fixed.